app/models/mixins/ownership_mixin.rb

```diff
@@ -48,6 +50,17 @@ module OwnershipMixin
 
       t.grouping(Arel::Nodes::NamedFunction.new("LOWER", [arel_attribute(:owning_ldap_group)]).eq(ldap_group))
     end)
+
+    def collection_name
+      collections = Api::ApiConfig[:collections].to_h.slice(*ALLOWED_COLLECTIONS)
```

Comment:
It would be good if this can be more dynamic, without having to add classes in the array constant.

Perhaps this can just be a class method as well. I would think we could look this up and memorize it as well.